toxcore/mono_time.h

```diff
@@ -45,8 +45,13 @@ extern "C" {
  */
 typedef struct Mono_Time Mono_Time;
 
-Mono_Time *mono_time_new(void);
-non_null() void mono_time_free(Mono_Time *mono_time);
+typedef uint64_t mono_time_current_time_cb(void *user_data);
+
+nullable(1, 2)
+Mono_Time *mono_time_new(mono_time_current_time_cb *current_time_callback, void *user_data);
+
+non_null()
+void mono_time_free(Mono_Time *mono_time);
```

Comment:
I'd expect `_free(...)` type functions to generally be nullable. Is this a conscious decision against this convention?

Reply:
Maybe not, but you tell me to stop doing multiple things per PR and then you tell me to change nullability of functions I'm not touching (only reformatting)?

Reply:
I'm ok with multiple things per PR, as long as they are separate commits or properly explained. I didn't quite see that this was only moved, but nevertheless it's strange and warants a comment. As agreed on IRC this will be globally changed.